web/src/lib/components/onboarding-page/onboarding-hello.svelte

```diff
@@ -1,28 +1,23 @@
 <script lang="ts">
   import ImmichLogo from '$lib/components/shared-components/immich-logo.svelte';
   import { user } from '$lib/stores/user.store';
-  import { Button } from '@immich/ui';
-  import { mdiArrowRight } from '@mdi/js';
   import { t } from 'svelte-i18n';
-  import OnboardingCard from './onboarding-card.svelte';
+  import { OnboardingRole } from '$lib/models/onboarding-role';
+  import { serverConfig } from '$lib/stores/server-config.store';
 
-  interface Props {
-    onDone: () => void;
-  }
+  let userRole = $derived($user.isAdmin && !$serverConfig.isOnboarded ? OnboardingRole.SERVER : OnboardingRole.USER);
 
-  let { onDone }: Props = $props();
+  export async function save() {}
```

Comment:
Why are you exporting this function here? In 99% the cases this is a bad smell and shouldn't be used in components.